nostr/src/commonMain/kotlin/work/socialhub/planetlink/nostr/action/NostrAction.kt

```
@@ -0,0 +1,545 @@
+                    createAt = Instant.fromEpochSeconds(dm.createdAt, 0)
+                    text = work.socialhub.planetlink.model.common.AttributedString.plain(dm.content)
+                    directMessage = true
+
+                    val authorPubkey = dm.senderPubkey
+                    if (authorPubkey == pubkey) {
+                        this.user = userMe
+                    } else {
+                        try {
+                            val profile = social.users().getProfile(authorPubkey)
+                            this.user = NostrMapper.user(profile.data, service())
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)